requirements.txt

```diff
@@ -0,0 +1,6 @@
+asgiref==3.9.1
+Django==4.0.2
+gunicorn
+sqlparse==0.5.3
+whitenoise
+pillow
```

Comment:
in the best case the versions should be pinned.

E.g. when using a 3.9 interpreter you could do the following: 

- build the container image
- start the container
- run `docker exec` to execute `pip freeze -l` within the container to obtain only the installed versions in the environment
- update your `requirements.txt`